tools/fast/bin/localnet/main.go

```diff
@@ -161,6 +162,11 @@ func main() {
 	}()
 
 	defer func() {
+		if r := recover(); r != nil {
+			fmt.Println("recovered from panic", r)
+			fmt.Println("stacktrace from panic: \n" + string(debug.Stack()))
+			os.Exit(exitcode)
```

Comment:
This line is redundant as is, but I think we want to either set the exitcode, or just explicitly exit with a 1.